infra/HOMELAB.md

```
@@ -0,0 +1,78 @@
+# homelab stack
+
+server: `10.0.0.5` (ssh semyon@server), Ubuntu 24.04, 30 GB RAM, 457 GB NVMe, GTX 1050 4 GB.
+
+## stack location
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)